API/Controllers/AquariumController.cs

```diff
@@ -27,5 +28,21 @@ public async Task<IActionResult> Get([FromRoute] int id)
             var aquariumViewModel = _mapper.Map<AquariumViewModel>(aquarium);
             return Ok(aquariumViewModel);
         }
+
+        [HttpDelete]
+        [Route("{id}")]
+        public async Task<IActionResult> Delete([FromRoute] int id)
+        {
+
+          var aquariumState =  await _aquariumService.Delete(id);
+
+
+          if(aquariumState)  return NoContent();
+
+
+          return NotFound();
```

Comment:
Nie ma potrzeby zwracanie notFound. Gdy nie ma takiego zasobu, to po prostu zwróć NoContent.